pkg/services/tree/sync.go

```diff
@@ -118,18 +140,20 @@ func (s *Service) SynchronizeTree(ctx context.Context, cid cid.ID, treeID string
 		return nil
 	}
 
-	return s.synchronizeTree(ctx, d, treeID, nodes)
+	_, err = s.synchronizeTree(ctx, d, 0, treeID, nodes)
+	return err
 }
 
-func (s *Service) synchronizeTree(ctx context.Context, d pilorama.CIDDescriptor,
-	treeID string, nodes []netmapSDK.NodeInfo) error {
-	lm, err := s.forest.TreeGetOpLog(d.CID, treeID, 0)
-	if err != nil && !errors.Is(err, pilorama.ErrTreeNotFound) {
-		return err
-	}
+func (s *Service) synchronizeTree(ctx context.Context, d pilorama.CIDDescriptor, from uint64,
+	treeID string, nodes []netmapSDK.NodeInfo) (uint64, error) {
```

Comment:
`error` looks useless now, i would remove it for readability

Reply:
Fixed